cmd/restic/cmd_diff.go

```diff
@@ -273,6 +274,14 @@ func (c *Comparer) diffTree(ctx context.Context, stats *DiffStatsContainer, pref
 				!reflect.DeepEqual(node1.Content, node2.Content) {
 				mod += "M"
 				stats.ChangedFiles++
+
+				node1NilContent := node1
```

Comment:
`node1` is a pointer, use `node1NilContent := *node1` to actually copy it.

Reply:
Oh. The only reason this didn't break anything is because there's no access to `.Content` later on, right?

Reply:
Exactly.